src/stock-adjustment-mobile/edit-product-page/edit-product-page.jsx

```diff
@@ -0,0 +1,315 @@
+                                                    disabled={true}
+                                                />
+                                                {renderLotSelect(name, values.items[index].product, values.items[index])}
+                                                <ReadOnlyField
+                                                    name="expiryDate"
+                                                    label="Expiry Date"
+                                                    containerClass='form-field-full-width'
+                                                    formatValue={formatDate}
+                                                />
+                                                <ReadOnlyField
+                                                    numeric
+                                                    name="stockOnHand"
+                                                    label="Stock on Hand"
+                                                    containerClass='field-full-width'
+                                                />
+                                                <SelectField
+                                                    required
+                                                    name={`${name}.reason`}
+                                                    label="Reason"
+                                                    options={reasons}
+                                                    objectKey="id"
+                                                />
+                                                <InputField
+                                                    required
+                                                    numeric
+                                                    name={`${name}.quantity`}
+                                                    label="Quantity"
+                                                    containerClass='field-full-width'
+                                                />
+                                            </div>
+                                        ))}
+                                    </div>
+                                    
+                                    <div className="navbar">
+                                        <div id='navbar-wrap'>
+                                        <button type="button" onClick={() => confirmAlertCustom({
+                                                title: "Are you sure you want to delete this product from Adjustments?",
+                                                confirmLabel: 'Delete',
+                                                confirmButtonClass: 'danger',
+                                                onConfirm: deleteProduct
+                                            })} 
+                                            className="danger"
+                                            style={{marginLeft: "5%"}}
+                                        >
```

Comment:
No need to make changes but it would be better if indentation was consistent - either all props in a single line or all props one beneath the other.